v22.1/export.md

```diff
@@ -73,6 +80,35 @@ Successful `EXPORT` returns a table of (perhaps multiple) files to which the dat
 `rows` | The number of rows exported to this file.
 `bytes` | The file size in bytes.
 
+## Parquet types
+
+CockroachDB types map to [Parquet types](https://github.com/apache/parquet-format/blob/master/LogicalTypes.md) as per the following:
+
+| CockroachDB Type    | Parquet Type | Parquet Logical Type |
+--------------------|--------------|----------------------
+| [`BOOL`](bool.html) | `BOOLEAN` | `nil` |
+| [`STRING`](string.html) | byte array | `STRING` |
+| [`COLLATE`](collate.html) | byte array | `STRING` |
+| [`INET`](inet.html) | byte array | `STRING` |
+| [`JSONB`](jsonb.html) | byte array | `JSON` |
+| [`INT`](int.html) | `INT64` | `nil` |
+| [`FLOAT`](float.html) | `FLOAT64` | `nil` |
+| [`DECIMAL`](decimal.html) | byte array | `DECIMAL` <br>Note: scale and precision data are preserved in the Parquet file |
+| [`UUID`](uuid.html) | `fixed_len_byte_array` | `nil` |
+| [`BYTES`](bytes.html) | byte array | `nil` |
+| [`BIT`](bit.html) | byte array | `nil` |
+| [`ENUM`](enum.html) | byte array | `ENUM` |
+| [`Box2D`](data-types.html#data-type-conversions-and-casts) | byte array | `STRING` |
+| [`GEOGRAPHY`](data-types.html#data-type-conversions-and-casts) | byte array | `nil` |
+| [`GEOMETRY`](data-types.html#data-type-conversions-and-casts) | byte array | `nil` |
+| [`DATE`](date.html) | byte array | `STRING` |
+| [`TIME`](time.html) | `INT64` | `TIME` <br>Note: microseconds after midnight |
+| [`TIMETZ`](time.html) | byte array | `STRING` |
```

Comment:
nit: it's probably worth clarifying that all time/timestamp types export to the precision of microseconds.